docs/profiling/algorithm_implementations.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
Github will [render this out](https://github.com/mongodb/signal-processing-algorithms/blob/DAG-100/docs/profiling/algorithm_implementations.ipynb) for us.